### ISSUE TITLE
How to finetune ARMO model with custom dataset?

### ISSUE THREAD

**Issue by Helen-Cheung** [RLHFlow/RLHF-Reward-Modeling#23]
How can I Fine-tuning the ARMO model with a custom dataset that only contains paired preference data without multi-objective reward scores？: )

**Comment by nshen7** [comment]
Was wondering about the same question!

**Comment by WeiXiongUST** [comment]
@Haoxiang-Wang hi haoxiang, can you take a look into this?

**Comment by Haoxiang-Wang** [comment]
@Helen-Cheung @nshen7 I will push the code soon. Stay tuned!

**Comment by Haoxiang-Wang** [comment]
Training code released!
